Ports/qemu/package.sh

```diff
@@ -3,17 +3,23 @@ port='qemu'
 version='7.2.0'
 useconfigure='true'
 configopts=(
-    "--target-list=aarch64-softmmu,i386-softmmu,x86_64-softmmu"
-    "--cross-prefix=${SERENITY_ARCH}-pc-serenity-"
-    "--extra-ldflags=-lm"
-    "--without-default-features"
-    "--disable-strip"
-    "--enable-pie"
-    "--enable-sdl"
-    "--enable-tcg"
-    "--enable-tools"
+    '--target-list=aarch64-softmmu,i386-softmmu,x86_64-softmmu'
+    '--cross-prefix=${SERENITY_ARCH}-pc-serenity-'
```

Comment:
This should be in double quotes for the interpolation, though.

Reply:
Thanks for noticing, Daniel! Things went about a bit too fast..

Additionally I reformatted the `configopts` section for `libslirp` to have each entry on a separate line.